### ISSUE TITLE
woocommerce rest api 401 error

### ISSUE THREAD

**Issue by Nick-tech-2691** [woocommerce/woocommerce#27584]
![post](https://user-images.githubusercontent.com/52854535/92202687-2702ea00-ee9d-11ea-91f5-41e8a42652b9.png)
this problem is coming with all the projects which are using woocommerce plugin, all of them were working fine but suddenly since this Monday, I'm stuck with this problem.
I've used insomnia also but the same thing is coming up.

**WordPress 5.4.2
WooCommerce plugin 4.4.1
Rest API v3**

**Comment by om4james** [comment]
Hi @Nick-tech-2691,

This could possibly be a duplicate of #26847 ?

**Comment by rodrigoprimo** [comment]
Hi @Nick-tech-2691,

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.

The type of issue you submitted looks like a support request which may or may not reveal a bug once proper troubleshooting is done. In order to confirm the bug, please follow one of the steps below:

- Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

If you confirm the bug, please provide us with clear steps to reproduce it.

I am closing this issue for now as it seems to be a support request and not a bug. If I missed something, please leave a comment and we will take a second look.

**Comment by dineufeld** [comment]
@SUNV1726 I've had the same problem since yesterday, it just suddenly came out of nowhere. How did you solve it?
